README.md

```diff
@@ -77,3 +77,40 @@ To review:
 2. Go through the code.
 3. If something is not clear, ask for clarification. If someone has already asked just thumbs up that comment.
 4. Add an approving comment saying "Good work!". If someone has already commented thumbs up on that.
+
+## How To Contribute and Run Software
+* Fork This Repo by clicking on button as shown in image.
+![Fork](screenshots/1.png)
+
+* now you sould have a copy of Repo for yourself with `username/pyjudge`.
+![Clone](screenshots/2.png)
+
+* after this you will need to clone from the Green tab you can see in above image.
+
+* now let's come to your local machine.
+    * open terminal
+    * create a folder "PyJudge" by `$ mkdir PyJudge`.
+    * go to folder `$ cd PyJudge`.
+    * initialize the git by ` $ git init`.
+    * `$ git clone https://github.com/YourUsername/PyJudge.git`
+    * now you have a folder called PyJudge which has all your code files.
+    * now go to that folder `cd PyJudge`.
+    * create a virtual enviroment `virtualenv venv`.
+    * activate your enviroment `source venv/bin/activate`.
+    * install dependencies `pip install -r requirements.txt`.
+    * and run the Software by `python server.py`.
+
+    ### To Contribute
+    * follow the above setup once.
+    *  create a new github branch in `/PyJudge/PyJudge` by `git checkout -b BranchName`
+    * now make changes in your files.
+    * when done see changes from `git status`. it will show all file which you updated or modified.
+    * add all those file to your git repo by `git add .`. `.` dot will add all files if you want a perticuler file to be added use, `git add filename`
```

Comment:
I recommend using `git add -Av` easier to catch errors than `git add .`

Reply:
do you mean `git add -A` ?